lib/govuk_app_config/govuk_prometheus_exporter.rb

```diff
@@ -1,14 +1,34 @@
 module GovukPrometheusExporter
   def self.configure
-    unless Rails.env == "test" || (ENV["GOVUK_PROMETHEUS_EXPORTER"]) != "true"
+    if (ENV["GOVUK_PROMETHEUS_EXPORTER"]) == "true"
+      return if defined?(Rails) ? Rails.env == "test" : false
+
       require "prometheus_exporter"
       require "prometheus_exporter/server"
       require "prometheus_exporter/middleware"
 
+      if defined?(Sidekiq)
+        Sidekiq.configure_server do |config|
+          require "prometheus_exporter/instrumentation"
+          config.server_middleware do |chain|
+            chain.add PrometheusExporter::Instrumentation::Sidekiq
+          end
+          config.death_handlers << PrometheusExporter::Instrumentation::Sidekiq.death_handler
+          config.on :startup do
+            PrometheusExporter::Instrumentation::Process.start type: "sidekiq"
```

Comment:
```suggestion
            PrometheusExporter::Instrumentation::Process.start(type: "sidekiq")
```
https://rubystyle.guide/#method-call-parentheses
(similarly throughout)